src/vs/workbench/contrib/extensions/electron-browser/extensionTipsService.ts

```diff
@@ -1138,7 +1136,71 @@ export class ExtensionTipsService extends Disposable implements IExtensionTipsSe
 			.map(extensionId => (<IExtensionRecommendation>{ extensionId, sources: ['application'] })));
 	}
 
+	promptVisualizerExtensions(): void {
+		const storageKey = 'extensionsAssistant/VisualizerRecommendationsIgnore';
+
+		if (this.storageService.getBoolean(storageKey, StorageScope.GLOBAL, false)) {
+			return;
+		}
+
+		let recommendations: IExtensionRecommendation[];
+		let localExtensions: ILocalExtension[];
+		const getRecommendationPromise = this.getVisualizerExtensions().then(recs => { recommendations = recs; });
+		const getLocalExtensionPromise = this.extensionsService.getInstalled(ExtensionType.User).then(local => { localExtensions = local; });
+		Promise.all([getRecommendationPromise, getLocalExtensionPromise]).then(() => {
+			if (!recommendations.every(rec => { return localExtensions.findIndex(local => local.identifier.id.toLocaleLowerCase() === rec.extensionId.toLocaleLowerCase()) !== -1; })) {
+				return new Promise<void>(c => {
+					this.notificationService.prompt(
+						Severity.Info,
+						localize('visualizerRecommendations.VisualizerExtensions', "Azure Data Studio has data visualization extension recommendations."),
+						[{
+							label: localize('visualizerRecommendations.installAll', "Install Extension"),
+							run: () => {
+								this.telemetryService.publicLog('visualizerRecommendations:popup', { userReaction: 'install' });
+								const installAllAction = this.instantiationService.createInstance(InstallVisualizerExtensionsAction, InstallVisualizerExtensionsAction.ID, localize('installAll', "Install All"), recommendations);
+								installAllAction.run();
+								installAllAction.dispose();
+								c(undefined);
+								const message = 'Your extension is ready. Select the Visualizer icon to visualize your data.';
+								this.notificationService.info(message);
+							}
+						}, {
+							label: localize('visualizerRecommendations.showMoreInfo', "More Info"),
```

Comment:
Shouldn't this be "View Extensions" or similar? this all needs review with Alan Ren, UX & PM so it's consistent with other recommendations

Reply:
Awesome, thanks!